scripts/configurator/parser/cmake_row.py

```diff
@@ -87,18 +62,16 @@ def is_pound_comment(self, row: str) -> bool:
       return row[0] == "#"
 
    def serialize(self) -> str:
-      serialized_val: str
       if self.row_type == row_type.VARIABLE:
-         if self.configurator_type == configurator_type.BOOL:
-            serialized_val = "ON" if self.val else "OFF"
-         else:
-            serialized_val = str(self.val)
-         return self.name + ":" + self._cmake_type_str + "="  + serialized_val
-      elif self.row_type == row_type.POUND_COMMENT:
+         return self.name + ":" + self._cmake_type + "=" + self.val.serialize()
+      elif type(self.val) == str and self.row_type == row_type.POUND_COMMENT:
          return "#" + self.val
-      elif self.row_type == row_type.SLASH_COMMENT:
+      elif type(self.val) == str and self.row_type == row_type.SLASH_COMMENT:
          return "//" + self.val
       return "" # in case of empty string
 
-   def normalize_bool(self, val: str) -> bool:
-      return self.NORMALIZED_BOOLS[val]
+   def get_val(self):
```

Comment:
I get what you'd like to have with this `get_val()`. I have a bit of mixed feelings, but I tend for keeping it at the moment, as it does a great job for the tests.

Reply:
Ok, perfect